src/schema/v2/partner.ts

```diff
@@ -74,6 +74,10 @@ const artworksArgs: GraphQLFieldConfigArgumentMap = {
     type: new GraphQLList(GraphQLString),
     description: "Return only artwork(s) included in this list of IDs.",
   },
+  artistID: {
```

Comment:
this should be an array similar to artworkIDs for future compatibility, thanks @mdole

Reply:
jk gravity only supports passing 1 to this endpoint